source/usage-examples/find.txt

```diff
@@ -4,13 +4,15 @@ Find Multiple Documents
 
 .. default-domain:: mongodb
 
-You can query for multiple documents in a collection with
-``collection.find()``. The ``find()`` method uses a query document that you
-provide to match the subset of the documents in the collection that match the
-query. If you don't provide a query document (or if you provide an empty
-document), MongoDB returns all documents in the collection. For more
-information on querying MongoDB, see our
-:doc:`documentation on query documents </fundamentals/crud/query-document>`.
+You can query for multiple documents in a collection with the
+``collection.find()`` method. The ``find()`` method uses a query
+document that contains one or more query operators to determine the
+documents to include in the result set. If you don't provide a query
+document or if you provide an empty document, MongoDB returns all
+documents in the collection. For more information on how to 
+specify query operators in a query document, see our
+:doc:`documentation on query documents
```

Comment:
```suggestion
:doc:`guide on query documents
```

Nit: I think in other parts of the drivers docs we refer to our own pages as guides. Also the phrase "documentation on query documents" sounds off to me with the double use of "document". 

Applies to both instances